cryptoki/tests/basic.rs

```diff
@@ -1268,6 +1268,28 @@ fn sha256_digest() -> TestResult {
     Ok(())
 }
 
+#[test]
+#[serial]
+fn gcm_param_graceful_failure() -> TestResult {
+    // Try to generate GcmParams with max size IV (2^32-1)
+    // Verify that the ulIvBits doesn't cause failover
+    println!("start");
+    // setting this as a [u8] array causes stack overflow before operation has even begun
+    let mut iv = vec![0; 4294967295];
+    println!("iv");
+    let aad = [0; 16];
+    
+    let _gcm_params = match GcmParams::new(&mut iv, &aad, 96.into()) {
+        Ok(params) => params,
+        Err(e) => { 
+            println!("{e}");
+            return Err(e.into());
+        },
+    };
```

Comment:
I don't think this entire thing is necessary. See if this is better:

```suggestion
    GcmParams::new(&mut iv, &aad, 96.into())?;
```